### PR TITLE
[graphicsmagick] Updated to GraphicsMagick-1.3.34

### PR DESCRIPTION
**Pull request by josuegomes**
Upgrade to Graphics 1.3.34

x86-windows
x64-windows
x86-windows-static
x64-windows-static
x64-linux

### CODE REVIEW

File: ports/graphicsmagick/portfile.cmake [hunk 1]
```diff
@@ -1,11 +1,11 @@
 include(vcpkg_common_functions)
```
Comment (NancyLi1013) [inline ports/graphicsmagick/portfile.cmake]: `include(vcpkg_common_functions)` is no longer needed.
Could you please remove this line?

File: ports/graphicsmagick/portfile.cmake [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-set(GM_VERSION 1.3.33)
+set(GM_VERSION 1.3.34)
 
 vcpkg_download_distfile(ARCHIVE
     URLS "https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/${GM_VERSION}/GraphicsMagick-${GM_VERSION}-windows-source.7z"
     FILENAME "GraphicsMagick-${GM_VERSION}-windows-source.7z"
-	SHA512 856fafe36693a7963f5944f489f4b7a0d15d9dcc575f96d31486e3abd886f681854cbb2271feb4b4cb19668480904d473138a2d9c2ec95539de705d75e2f49c9
+	SHA512 bc770070d3e6b58ded1932050c1aa9f31edd89ac68e12055d7bb481ba9aa8e7509e69ad676a1624c262f9b55c0ceeea738b807005946ddd97b88d49b39b8a073
 )
```
Comment (NancyLi1013) [inline ports/graphicsmagick/portfile.cmake]: Could you update this 
```
file(COPY ${SOURCE_PATH}/Copyright.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/graphicsmagick)
file(RENAME ${CURRENT_PACKAGES_DIR}/share/graphicsmagick/Copyright.txt ${CURRENT_PACKAGES_DIR}/share/graphicsmagick/copyright)
```
as `file(INSTALL ${SOURCE_PATH}/Copyright.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)`?

In `copy config` field, please also update `${CURRENT_PACKAGES_DIR}/share/graphicsmagick/config` as
`${CURRENT_PACKAGES_DIR}/share/${PORT}/config`.

Comment (NancyLi1013) [reply ports/graphicsmagick/portfile.cmake]: `vcpkg_test_cmake()` is deprecated, could you please remove `vcpkg_test_cmake(PACKAGE_NAME unofficial-graphicsmagick)`?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Review by PhoebeHui — APPROVED** [review]

**Pull request merged**